pkg/apihelpers/apihelpers.go

```diff
@@ -23,6 +25,17 @@ func NewMachineConfigPoolCondition(condType mcfgv1.MachineConfigPoolConditionTyp
 	}
 }
 
+// NewMachineConfigPoolCondition creates a new MachineConfigPool condition.
+func NewMachineOSBuildCondition(condType string, status metav1.ConditionStatus, reason, message string) *metav1.Condition {
+	return &metav1.Condition{
```

Comment:
thought: Would it make sense to put these helpers in a separate file called, e.g., `machineosbuildcondition.go`? This would be purely for organizational purposes.